MAINTAINERS.md

```diff
@@ -10,3 +10,4 @@ This file lists who are the core maintainers of the TiKV project.
 
 * [Jay Li](https://github.com/busyjay); jay@pingcap.com; Focus Areas: Raft, Coprocessor
 * [Jinpeng Zhang](https://github.com/zhangjinpeng1987); zhangjinpeng@pingcap.com; Focus Areas: Storage, Transaction
+* [Xiaoguang Sun](https://github.com/sunxiaoguang); sunxiaoguang@zhihu.com; Focus Areas: Storage, Search Engine
```

Comment:
Congratulations! BTW Focus area should be the role that you involve in the TiKV project.

Reply:
I see. Let me fix it.